hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/interfaces/ContainerDispatcher.java

```diff
@@ -93,4 +94,15 @@ default StateMachine.DataChannel getStreamDataChannel(
     throw new UnsupportedOperationException(
         "getStreamDataChannel not supported.");
   }
+
+  /**
+   * When reading data form client using stream, get StreamDataChannel.
+   */
+  default void streamDataReadOnly(
+       ContainerCommandRequestProto msg,
+       StreamObserver<ContainerCommandResponseProto> streamObserver,
+       DispatcherContext dispatcherContext) {
+    throw new UnsupportedOperationException(
+         "getStreamDataChannel not supported.");
```

Comment:
This "getStreamDataChannel not supported."  uses the incorrect function name.

Reply:
Done